contour/how-to-guides/configuring-contour-with-loadbalancer-tls-termination.hbs.md

```diff
@@ -56,6 +111,26 @@ The following steps correspond to the steps in the [Contour open source document
 
```

Comment:
the `-f tap-values.yaml` in the tanzu package installed update command should be changed to `--values-file tap-values.yml`